modules/integrating-loki-network-policy-with-external-systems.adoc

```diff
@@ -0,0 +1,52 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-10-17
+:_mod-docs-content-type: PROCEDURE
+
+[id="integrating-loki-network-policy-with-external-systems_{context}"]
+= Integrating Loki network policy with external systems
+
+To integrate Loki with external systems such as custom dashboards, external alerting, and so on, create additional network policies.
+You can select specific components by using the label `app.kubernetes.io/component`.
+Always include the labels `app.kubernetes.io/name=lokistack` and `app.kubernetes.io/instance={name}` to avoid collision with other pods deployed in the namespace.
+
+.Prerequisites
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {loki-op}.
+* You have created a `LokiStack` custom resource (CR).
+
+.Procedure
+. Create a network policy:
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: <name>
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: lokistack
+      app.kubernetes.io/instance: <instance_name>
+      app.kubernetes.io/component: <loki_component>
+  policyTypes:
+  - Egress
+  egress:
+  - to:
+    - namespaceSelector:
```

Comment:
Suggest to make this config as below. Namespaces do not have` name:` labels automatically
```
namespaceSelector:
  matchLabels:
    kubernetes.io/metadata.name: <namespace_name>
```